src/style.css

```diff
@@ -0,0 +1,142 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+:root {
+  --main-accent-color: #060b35;
+  --secondary-color: #616070;
+  --button-background-color: #00acdc;
+  --primary-background-color: #fff;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: var(--primary-background-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  margin: 0 auto;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-top: 40px;
+  font-family: inherit;
```

Comment:
No needs to inherit this rule everywhere. It is cascading form parent element/ 
Check all over the page

```suggestion

```